### ISSUE TITLE
EXASOL support

### ISSUE THREAD

**Issue by nooce** [ultorg/public_issues#42]
Greetings,
I'd like to use Ultorg as a front end for EXASOL.
I succeed to register a database connection as a service.
As soon as I try to access to the data I get the message
INFO [com.ultorg.project.result.e]: Unexpected problem during query evaluation
com.ultorg.util.UnexpectedException: markCollation did not return a TextStorageType

Do you have any track to solve this issue

Regards

**Comment by eirikbakke** [comment]
Exasol connections aren't officially supported by Ultorg, though I did manage to connect to their trial instance. There are a few immediate errors like this which I've now fixed, for the next Ultorg release. I'll let you know when a new version is available!

What size is the data you're working with? (E.g. number of rows per table?)

**Comment by nooce** [comment]
Hi Eirik,

Size of the dataset : it depends on the customer, most of the time between 200 millions rows and 2 billions, biggest was 70 billions.
A model contains between 10 and 100 tables




Le 10 oct. 2022 à 23:16, Eirik Bakke ***@***.***> a écrit :

﻿

Exasol connections aren't officially supported by Ultorg, though I did manage to connect to their trial instance. There are a few immediate errors like this which I've now fixed, for the next Ultorg release. I'll let you know when a new version is available!

What size is the data you're working with? (E.g. number of rows per table?)
